modules/k8s-cluster/main.tf

```diff
@@ -0,0 +1,20 @@
+resource "null_resource" "create" {
+    count = var.command == "create" ? 1 : 0
+  provisioner "local-exec" {
```

Comment:
You can do this with `when`, not sure how exactly, but should be possible.
See https://www.terraform.io/docs/language/resources/provisioners/syntax.html#destroy-time-provisioners

Reply:
vor bash scripti mej popoxakan em oktagorcum espisi erora tali.
u dramasin terrafromy xosuma estex 
https://www.terraform.io/upgrade-guides/0-13.html#destroy-time-provisioners-may-not-refer-to-other-resources
```
Error: Invalid reference from destroy provisioner

Destroy-time provisioners and their connection configurations may only
reference attributes of the related resource, via 'self', 'count.index',
or 'each.key'.

References to other resources during the destroy phase can cause dependency
cycles and interact poorly with create_before_destroy.
```

Reply:
Not clear what is the issue here. Let's discuss this in person tomorrow.